test/conftest.py

```diff
@@ -48,6 +48,7 @@ def vcr(vcr):
     vcr.filter_post_data_parameters = POST_DATA_BLACKLIST
     vcr.before_record_request = scrub_request(CLIENT_INIT)
     vcr.before_record_response = scrub_response(RESPONSE_BLACKLIST)
+    vcr.decode_compressed_response = True
```

Comment:
wild this isn't default - I'd assume that you want this 99% of the time?
The docs say 


> This option should be avoided if the actual decompression of response bodies is part of the functionality of the library or app being tested.

but how often are you actually doing that yourself?

Reply:
assuming it's the gzip decompression right?

Reply:
Yep, gzip compression. At some point Welkin started returning responses like this.